ibm/mas_devops/roles/cp4d_wds/README.md

```diff
@@ -0,0 +1,98 @@
+cp4d_wds
+==========
+
+It's only for WDS 4.0.x install on the existing CP4D 4.0.x and WDS instance provisioning, and also generated WDScfg (For Assist Install) related yaml to MAS config Directory.
+
+It's also used for WDScfg (For Assist Install) related yaml preparation if you want to use the existing external WDS instance. 
+
+Role Variables
+--------------
+### wdschannel
+The discovery channel used by discovery install. By default the `v4.0`  will be used as channel name for discovery install.  
+- Environment Variable: `WDS_CHANNEL`
+- Default Value: v4.0
+
+### wdsversion
+The discovery version used by discovery install. By default the discovery `4.0.4` version will be installed. 
+- Environment Variable: `WDS_CHANNEL`
+- Default Value: 4.0.4
+
+### wdsstorageclass
+The specific storage class for discovery install, if not specified , the storage class used by CP4D will be auto queried in the cluster and used for discovery.
+install.
```

Comment:
"install" should be at the end of the previous line?

Reply:
fixed